resources/styles/molecules/tabs/BpTabs.vue

```diff
@@ -96,117 +100,143 @@ export default {
         },
         wrap: {
             type: Boolean,
-            default() {
-                return !this.vertical;
-            },
         }
-    },
+    })
 
-    data () {
-        return {
-            activeTab: '',
-            currentFocusTab: '',
-        }
-    },
+    //const { isActive } = useKeyboard(props.tabs, activeTab)
 
-    computed: {
-        tabKeys () {
-            return Object.keys(this.tabs)
-        },
-        firstTabKey () {
-            return this.tabKeys[0]
-        },
-        lastTabKey () {
-            return this.tabKeys[this.tabKeys.length - 1]
-        },
-        currentFocusTabIndex () {
-            return this.tabKeys.indexOf(this.currentFocusTab)
-        },
-        nextTabKey () {
-            return this.currentFocusTab === this.lastTabKey
-                ? (this.wrap ? this.firstTabKey : null)
-                : this.tabKeys[this.currentFocusTabIndex + 1]
-        },
-        prevTabKey () {
-            return this.currentFocusTabIndex === 0
-                ? (this.wrap ? this.lastTabKey : null)
-                : this.tabKeys[this.currentFocusTabIndex - 1]
-        },
-    },
-
-    mounted () {
-        this.setInitFocusTab()
-        this.setInitActiveTab()
-        this.attachKeyboardListeners()
-    },
-
-    methods: {
-        setInitActiveTab () {
-            if (location.hash && this.tabKeys.indexOf(location.hash.substr(1)) !== -1) {
-                this.activeTab = location.hash.substr(1)
-            } else if (this.initialTab) {
-                this.activeTab = this.initialTab
+    const initialTabComp = computed({
```

Comment:
Similar to below, I'm not sure you need this computed property. Instead, you can set it as a default value when defining props:

``` 
default (props) {
       const [firstTab] = Object.keys(props.tabs)
        return firstTab
 },
```